api/database/migrations/2020_08_05_135254_create_jwt_refresh_tokens_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateJwtRefreshTokensTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('jwt_refresh_tokens', function (Blueprint $table) {
+            $table->id();
+            $table->timestamps();
+            $table->string('token');
+            $table->unsignedBigInteger('user_id');
+
```

Comment:
I forgot to mention that an expiration date should be present here. I have updated the Trello ticket, please, add it too.